scripts/repos.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+# repos.sh is supposed to be downloaded and used from the umbrella directory
+# holding the OpenDevStack repositories. It will ensure all required
+# repositories are present on the local machine, and checked out at the
+# specified Git ref. The Git ref has to exist in the specified source project.
+# The source project defaults to the "opendevstack" organisation in GitHub, but
+# it is also possible to use this script to just fetch what is present in
+# Bitbucket (e.g. for admins wanting to work against an existing, possibly
+# customized installation of OpenDevStack without updating / changing it).
+
+set -ue
+
+WORKING_DIR=$(pwd)
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+if [[ "$SCRIPT_DIR" == *ods-core/scripts ]]; then
+    echo "This script must not be used from within ods-core."
+    echo "It is supposed to be downloaded to the umbrella directory holding all ODS respositories."
+    echo "The following example shows how to do this for the 'master' version:"
+    echo ""
+    echo "  UMBRELLA_DIR=~/opendevstack"
+    echo "  GIT_REF=master"
+    echo "  mkdir -p \$UMBRELLA_DIR && cd \$UMBRELLA_DIR"
+    echo "  curl -LO https://raw.githubusercontent.com/opendevstack/ods-core/\$GIT_REF/scripts/repos.sh"
+    echo "  chmod +x repos.sh"
+    echo "  ./repos.sh --git-ref \$GIT_REF"
+    echo ""
+    exit 1
+fi
+
+echo_done(){
```

Comment:
Could add a comment that since this is used for bootstrapping it is not using colored_output.sh

Reply:
Added as a comment in 10d7e94.